beacon_node/network/src/metrics.rs

```diff
@@ -297,13 +297,15 @@ lazy_static! {
     /*
      * Block Delay Metrics
      */
-    pub static ref BEACON_BLOCK_GOSSIP_PROPAGATION_VERIFICATION_DELAY_TIME: Result<Histogram> = try_create_histogram(
+    pub static ref BEACON_BLOCK_GOSSIP_PROPAGATION_VERIFICATION_DELAY_TIME: Result<Histogram> = try_create_histogram_with_buckets(
         "beacon_block_gossip_propagation_verification_delay_time",
         "Duration between when the block is received and when it is verified for propagation.",
+        linear_buckets(12_f64,12_f64,8)
```

Comment:
expected value for this should be around 5ms

Reply:
I adjusted the value. I will run sync a node to check

Reply:
I updated the values. I will try to sync a node and see how it looks.

Reply:
nts: looks good